ios_pod/Podfile

```diff
@@ -14,6 +14,7 @@ target 'GetPods' do
   pod 'Firebase/Functions', '8.8.0'
   pod 'Firebase/Installations', '8.8.0'
   pod 'Firebase/Messaging', '8.8.0'
+  pod 'Firebase/Performance', '8.8.0'
```

Comment:
Can you update this to the latest version, 8.9.1? (also resolve the merge conflict in this file)